ajax/libs/PKI.js/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "1.2.1",
   "repository": {
     "type": "git",
-    "url": "git://github.com/GlobalSign/PKI.js.git"
+    "url": "git://github.com/PeculiarVentures/PKI.js.git"
   },
   "homepage": "http://pkijs.org",
   "keywords": [
```

Comment:
How about also update the keywords, to let it as same with upstream.
ref: https://github.com/PeculiarVentures/PKI.js/blob/master/package.json#L54-L67